File: common/retex/src/commonMain/kotlin/com/mshdabiola/retex/aimplementation/parser/ParserFactoryAndroid.java
package com.mshdabiola.retex.aimplementation.parser;

import com.himamis.retex.renderer.share.platform.parser.Parser;
import com.himamis.retex.renderer.share.platform.parser.ParserFactory;

public class ParserFactoryAndroid extends ParserFactory {

    @Override
    public Parser createParser() {
        return new ParserA();
    }

}

File: modules/retex/src/main/kotlin/com/himamis/retex/renderer/android/parser/ParserFactoryAndroid.kt
package com.himamis.retex.renderer.android.parser

import com.himamis.retex.renderer.share.platform.parser.Parser
import com.himamis.retex.renderer.share.platform.parser.ParserFactory

class ParserFactoryAndroid : ParserFactory() {
    override fun createParser(): Parser {
        return ParserA()
    }
}
